RobotCode/src/main/java/frc/robot/Ports.java

```diff
@@ -0,0 +1,6 @@
+package frc.robot;
+
+public class Ports
+{
+
```

Comment:
No ports given for motors, need to add this, even just putting in "0" as the value